EntAppProject/src/main/java/com/jobook/EntAppProject/model/Review.java

```diff
@@ -1,55 +1,54 @@
-package com.jobook.EntAppProject.model;
-
-public class Review {
-	private Integer employeeId, customerId, jobId;
-	private String description, title;
-	private Double rating;
-	
-	public Review(Integer employeeId, Integer customerId, Integer jobId, String description, String title, Double rating) {
-		super();

```

Comment:
As the pull request comment mentions, having `super()` in a class that only extends `Object` by default is unnecessary as that initialization/association is created automatically when the program is run. This change is present in many of the models.